docs/semgrep-assistant/overview.md

```diff
@@ -49,42 +49,42 @@ By categorizing your code through component tags, Semgrep Assistant can help you
 
 ![Semgrep Assistant Component tag list](/img/assistant-component-tags.png)
 
+Component tags are available in Semgrep Cloud Platform's **Findings page**.
+
 ### Auto-triage
 
-Semgrep Assistant uses GPT-4's understanding of programming languages and libraries and your code and triage history to auto-triage findings and indicate if a finding can safely be ignored. For every recommendation to ignore a finding, Semgrep also provides guidance with an explanation on why this is the case.
+Semgrep Assistant uses GPT-4's understanding of programming languages and libraries, and your code and triage history, to auto-triage findings and suggest whether a finding can safely be ignored. For every recommendation to ignore a finding, Semgrep also provides guidance with an explanation on why this is the case.
 
-Semgrep presents these recommendations through Slack and PR or MR comments, so developers can fix or triage an issue without switching contexts.
+Auto-triage recommendations are available in Semgrep Cloud Platform's main **Findings page** when filtering to findings that Assistant suggests should be ignored, and in the [finding's details](/docs/semgrep-code/findings/#view-findings-details-about-a-specific-finding). 
 
-![Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
-    *Figure*. Semgrep Assistant in a Slack notification.
+![Semgrep Assistant in the filtered Findings page](/img/semgrep-assistant-autotriage-findings.png)
+
+Assistant's suggestions to ignore findings are also sent through Slack and PR or MR comments, so developers can triage an issue without switching contexts.
 
-You can view findings that Assistant suggests should be **Ignored** in Semgrep Cloud Platform's **Findings page**. <!-- #10438 -->
+![Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
+    *Figure*. Semgrep Assistant auto-triage in a Slack notification.
 
 ### Autofix Semgrep Code findings
 
-Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings identified using rules that don't have human-written autofix suggestions when it identifies a true positive. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, then Assistant will recommend its use in the autofix suggestion for the issue in your code.
+Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive, if the rule doesn't have a human-written autofix. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, Assistant can recommend its use in the autofix suggestion for the issue in your code.
```

Comment:
```suggestion
Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive if the rule doesn't have a human-written autofix. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, Assistant can recommend its use in the autofix suggestion for the issue in your code.
```

Reply:
Hmm yeah, on its own that comma is definitely wrong, thank you. :) I don't love how the sentence sounds with no commas, though; maybe it needs to be split up properly. What do you think of:

> Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive. Assistant will only suggest an autofix if the rule doesn't have a human-written autofix.

Reply:
Yes, that looks good!